src/api/hooks/useFetchProject.ts

```diff
@@ -1,14 +1,17 @@
 import { useQuery } from '@tanstack/react-query';
 import { useTranslation } from 'react-i18next';
+import { getIdentifierFromId } from '../../utils/general-helpers';
 import { fetchProject } from '../cristinApi';
 
 export const useFetchProject = (projectId: string) => {
   const { t } = useTranslation();
 
+  const projectIdentifier = !isNaN(Number(projectId)) ? projectId : getIdentifierFromId(projectId);
```

Comment:
```suggestion
  const projectIdentifier = isNaN(Number(projectId)) ? getIdentifierFromId(projectId) : projectId;
```

Invertere, slik at man slipper utropstegn? "Is not not a number" 😅 Eventuelt sjekke på annen type